packages/common/src/events/api/orval/api/orval-events.ts

```diff
@@ -388,15 +366,67 @@ export interface ICreateOnlineMeeting {
   url: string;
 }
 
+export type ICreateLocationGeometriesItem = { [key: string]: any };
```

Comment:
I find it a little strange/confusing to have `Item` in the name of this interface and `ILocationGeometriesItem`. Would it be more intuitive to name these something like `*Collection` vs `*Item`? Unsure if this would require API changes and/or complexity when updating the interfaces in the future given that we're using a generated client.

Reply:
This is another byproduct of the type generator. Maybe we can customize the suffix, I'll check. But also this will be more explicit with my next task